### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by dipen2000**
I created a product listing page for the e-commerce project , please check it out and review it. give your feedback.

### CODE REVIEW

File: CSS/reset.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    line-height: 1.5;
+    font-size: 1.6rem;
+    background-color: var(--background-color);
+}
+
+p,
+h1,
+h2,
+h3,
+h4,
```
Comment (dev-ashishsethi) [inline CSS/reset.css:21]: could have done in one line

File: CSS/styles.css [hunk 2]
```diff
@@ -1,2 +1,191 @@
...
+.highlight{
+    background-color : var(--light-test-color);
+    /* color : white; */
+    padding : 0.5rem;
+    border-radius: 5px;
+}
+
+
+.container{
+    /* border : 3px solid red;  */
```
Comment (dev-ashishsethi) [inline CSS/styles.css:47]: don't comment code

**Review by dev-ashishsethi — APPROVED** [review]
good to go

File: pages/product/product.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    justify-content: space-between;
+}
+
+.label{
+    font-size : 2rem;
+    font-family: var(--nav-list-items-font);
+}
+
+.product-listing-container{
+    /* border : 3px solid red; */
```
Comment (ShrutiLakhani) [inline pages/product/product.css:50]: Please remove commented code in case you dont need it anymore

File: components/carousel/carousel.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    box-shadow: none;
+}
+
+.carousel-overlay-container{
+    display : flex;
+    flex-direction: column;
+    /* border : 3px solid red; */
+    align-items: center;
+    justify-content : center;
+    border-radius: 15px;
```
Comment (ShrutiLakhani) [inline components/carousel/carousel.css:23]: Avoid using px over 10px

**Review by ShrutiLakhani — APPROVED** [review]
Good to go. Please look into suggested changes

File: CSS/styles.css [hunk 5]
```diff
@@ -1,2 +1,191 @@
...
+    width : 90%;
+    /* max-width : 130rem; */
+    margin : auto;
+    position : relative;
+    /* border : 3px solid red; */
+}
+
+/****************************** navigation bar  ***********************/
+
+nav{
```
Comment (ashishkatiyar2404) [inline CSS/styles.css:57]: use class instead of tag

File: CSS/styles.css [hunk 6]
```diff
@@ -1,2 +1,191 @@
...
+    align-items : center;
+} 
+
+.nav-search-bar{
+    width : 40rem;
+    padding : 0.8rem 0.8rem;
+    font-size: 1.5rem;
+    border-radius: 0.5rem;
+    background-color: lightgray;
+    /* border : 2px solid var(--stone-500); */
```
Comment (ashishkatiyar2404) [inline CSS/styles.css:96]: remove comments

File: pages/product/product.css [hunk 7]
```diff
@@ -0,0 +1,60 @@
+.content-section-container{
+    /* border : 3px solid red; */
+    display : grid;
+    grid-template-areas: "sidebar main-content";
+    grid-template-columns: 30rem minmax(0,1fr);
```
Comment (ashishkatiyar2404) [inline pages/product/product.css:5]: good to see using min-max function😀

File: pages/product/product.css [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+.label{
+    font-size : 2rem;
+    font-family: var(--nav-list-items-font);
+}
+
+.product-listing-container{
+    /* border : 3px solid red; */
+    padding : 1.5rem;
+    background-color: rgba(128, 128, 128, 0.288);
```
Comment (ashishkatiyar2404) [inline pages/product/product.css:52]: use root variables for colors

**Review by ashishkatiyar2404 — APPROVED** [review]
suggested some changes pls work on tht
rest code looks good to me 💯

**Pull request merged**